content/en/logs/guide/lambda-logs-collection-troubleshooting-guide.md

```diff
@@ -0,0 +1,118 @@
+
+{{< img src="logs/guide/lambda-monitoring-tab.png" alt="Monitoring tab"  style="width:80%;" >}}
+
+2. The monitoring tab displays a series of graphs indicating the following information about your Lambda function: 
+    * invocations
+    * errors
+    * logs
+
+3. If you don't see any data points on the "Invocations" graph, there may be a problem with the triggers you set for your function. See [Manage Your Function Triggers](#manage-your-function-triggers). To get insight into your lambda invocations without using the monitoring tab, see [Viewing Lambda metrics in Datadog](#viewing-lambda-metrics-in-datadog).
+4. If you see data points on the "Error count and success rate" graph, [Check the Lambda function logs](#check-the-lambda-function-logs) to see what error messages are being reported.
+
+## Viewing Lambda metrics in Datadog
+
+If you have enabled AWS Lambda metrics, you can view metrics related to Lambda invocations and errors within Datadog. The following metrics are all tagged with a key value `functionname` tag: 
+
+* `aws.lambda.invocations ` : Count of times the Lambda function has been triggered / invoked
+* `aws.lambda.errors ` : Count of errors that occurred when the function was invoked
+* `aws.lambda.duration ` : Average amount of time (in milliseconds) that it took for the lambda function to finish executing
+* `aws.lambda.duration.maximum` : Maximum amount of time (in milliseconds) that it took for the lambda function to finish executing
+* `aws.lambda.throttles` : Count of invocation attempts that were throttled due to invocation rates exceeding customer limits
+
+For more information on these and other AWS lambda metrics, see [Amazon Lambda Metrics][6].
+
+## Check the Lambda function logs
+
+From the monitoring tab, click "View logs in Cloudwatch".
+
+{{< img src="logs/guide/lambda-logs-cloudwatch.png" alt="Lambda logs in Cloudwatch"  style="width:80%;" >}}
+
+### API key
+
+If you see one of the following log lines it means that your API key isn't set correctly.
+
+```
+module initialization error: Missing Datadog API key
+```
+This means you didn't set the API key. Places you can set the API key are:
+* Directly in the Python code of your Lambda function
+* As an environment variable under the name `DD_API_KEY`
+* As a KMS Encrypted key under the name `DD_KMS_API_KEY`
+
+```
+module initialization error: The API key is not valid.
+```
+The API key you provided doesn't match any key Datadog recognizes. Double-check the spelling and make sure that your key corresponds to the org you are sending data to. If your org is in Europe, you need to specify a `DD_SITE` variable with the value `datadoghq.eu` in order for the API key to be properly matched with your account.
+
+```
+module initialization error: The API key is not the expected length. Please confirm that your API key is correct
+```
+The API key is either too short or too long. Double-check that you copied it over correctly.
+
+## Manage your function triggers
```

Comment:
Seems to be redundant with the above section where we ask to check the monitoring tab.

Should we package those together?